Shopify/ViewModel/HomeViewModel/HomeViewModel.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  HomeViewModel.swift
+//  Shopify
+//
+//  Created by Mahmoud on 25/02/2023.
+//
+
+import Foundation
+class HomeViewModel{
```

Comment:
that's what i've mentioned before 
the view model named and created  by it's functionality not  by where we'll use it